src/dashboard/src/components/archival_storage/views.py

```diff
@@ -106,74 +106,82 @@ def search(request):
             # for an attribute of an AIP, the aipfile must index that
             # information about their AIP in
             # elasticSearchFunctions.index_mets_file_metadata
-            # Because we're searching aips/aipfile and not aips/aip,
-            # cannot using LazyPagedSequence
             results = conn.search(
                 body=query,
                 index='aips',
                 doc_type='aipfile',
-                fields='AIPUUID,sipName',
                 sort='sipName:desc',
             )
+            # Given these AIP UUIDs, now fetch the actual information we want from aips/aip
+            buckets = results['aggregations']['aip_uuids']['buckets']
+            uuids = [bucket['key'] for bucket in buckets]
+            uuid_file_counts = {bucket['key']: bucket['doc_count'] for bucket in buckets}
+            query = {
+                'query': {
+                    'terms': {
+                        'uuid': uuids,
+                    },
+                },
+            }
+            index = 'aips'
+            doc_type = 'aip'
+            kwargs = {
+                'fields': 'name,uuid,size,created,status,AICID,isPartOf,countAIPsinAIC',
+                'sort': 'name:desc',
+            }
         else:
-            # To reduce amount of data fetched from ES, use LazyPagedSequence
-            def es_pager(page, page_size):
-                """
-                Fetch one page of normalized aipfile entries from Elasticsearch.
-
-                :param page: 1-indexed page to fetch
-                :param page_size: Number of entries on a page
-                :return: List of dicts for each entry with additional information
-                """
-                start = (page - 1) * page_size
-                results = conn.search(
-                    body=query,
-                    index='aips',
-                    doc_type='aipfile',
-                    from_=start,
-                    size=page_size,
-                    fields='AIPUUID,filePath,FILEUUID',
-                    sort='sipName:desc',
-                )
+            index = 'aips'
+            doc_type = 'aipfile'
+            kwargs = {
+                'fields': 'AIPUUID,filePath,FILEUUID',
+                'sort': 'sipName:desc',
+            }
+
+        # To reduce amount of data fetched from ES, use LazyPagedSequence
+        def es_pager(page, page_size):
+            """
+            Fetch one page of normalized aipfile entries from Elasticsearch.
+
+            :param page: 1-indexed page to fetch
+            :param page_size: Number of entries on a page
+            :return: List of dicts for each entry with additional information
+            """
+            start = (page - 1) * page_size
+            results = conn.search(
+                body=query,
+                from_=start,
+                size=page_size,
+                index=index,
+                doc_type=doc_type,
+                **kwargs
```

Comment:
They differ between the two code paths; rather than assign to two variables I stuck them in a dictionary.